ccl/streamingccl/streamingest: TestStreamingReplanOnLag failed

ccl/streamingccl/streamingest.TestStreamingReplanOnLag [failed](https://tanzanite.cluster.engflow.com/invocations/default/f14722b0-4357-4626-937e-83d85fe53581?testReportRun=1&testReportShard=15&testReportAttempt=1#targets-Ly9wa2cvY2NsL3N0cmVhbWluZ2NjbC9zdHJlYW1pbmdlc3Q6c3RyZWFtaW5nZXN0X3Rlc3Q=) on master @ [a2f1f379ee52ceee2b2aa6769f1fa162f8d6b8a7](https://github.com/cockroachdb/cockroach/commits/a2f1f379ee52ceee2b2aa6769f1fa162f8d6b8a7):


```
=== RUN   TestStreamingReplanOnLag
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestStreamingReplanOnLag4262448888
    test_log_scope.go:81: use -show-logs to present logs inline
    testutils.go:488: condition failed to evaluate within 45s: from testutils.go:521: waiting for stream ingestion job progress 1710827875.000000000,0 to advance beyond 1710827877.574577007,0
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestStreamingReplanOnLag4262448888
--- FAIL: TestStreamingReplanOnLag (53.50s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=15</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/disaster-recovery
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamingReplanOnLag.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-36820

ccl/streamingccl/streamingest.TestStreamingReplanOnLag [failed](https://tanzanite.cluster.engflow.com/invocations/default/efc8a553-d34b-4b19-ae9c-5efa8025afc8?testReportRun=1&testReportShard=15&testReportAttempt=1#targets-Ly9wa2cvY2NsL3N0cmVhbWluZ2NjbC9zdHJlYW1pbmdlc3Q6c3RyZWFtaW5nZXN0X3Rlc3Q=) on master @ [f0116ea373a2b87155e7f0264df4f783ce177360](https://github.com/cockroachdb/cockroach/commits/f0116ea373a2b87155e7f0264df4f783ce177360):


```
=== RUN   TestStreamingReplanOnLag
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestStreamingReplanOnLag964593691
    test_log_scope.go:81: use -show-logs to present logs inline
    testutils.go:488: condition failed to evaluate within 45s: from testutils.go:521: waiting for stream ingestion job progress 1710915230.000000000,0 to advance beyond 1710915230.064217292,0
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestStreamingReplanOnLag964593691
--- FAIL: TestStreamingReplanOnLag (53.39s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=15</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStreamingReplanOnLag.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go
+++ b/pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go
@@ -855,8 +855,6 @@ func TestStreamingReplanOnLag(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	skip.WithIssue(t, 120688)
-
 	skip.UnderDuressWithIssue(t, 115850, "time to scatter ranges takes too long under duress")
 	skip.UnderMetamorphic(t, "time to scatter ranges takes too long under non-default settings")
 

